client/layout/guided-tours/style.scss

```diff
@@ -124,11 +124,16 @@
 	.button {
 		min-width: 48%;
 	}
-	.guided-tours__quit-button {
+	.guided-tours__quit-button,
+	a.button {
```

Comment:
I know this wasn't introduced in this PR and suppose it's consistent with other tour-popups but feels odd we have custom styles like this outside the regular button style semantics & component.

<img width="298" alt="image" src="https://user-images.githubusercontent.com/87168/56028742-7503d480-5d21-11e9-8de3-b194807e8262.png">

I was expecting to see "text button": https://murieldesignsystem.blog/components/buttons/#types-and-anatomy

Reply:
I agree with @simison - if we can stay in line with our existing design patterns unless its absolutely necessary to deviate. Good eye!

Reply:
I agree with you folks, but what we're doing in this PR is just reusing what is already there. The only change I'm doing here is actually stying that button, because before that it was appearing with white background and white text color. I prefer not changing anything that touches the existing guided tours in this PR. Does that make sense to you @simison and @jeffgolenski?

Reply:
If we decide to change this, it will make sense to audit all existing guided tours. Let's leave that for another PR, cc @Automattic/onboarding-dev and @Automattic/onboarding-design